src/Target.h

```diff
@@ -139,7 +141,7 @@ struct Target {
      * OpenGL, because it is not capable of gpgpu, and is not
      * scheduled via Func::gpu_tile. */
     bool has_gpu_feature() const {
-        return has_feature(CUDA) || has_feature(OpenCL);
+        return has_feature(CUDA) || has_feature(OpenCL) || has_feature(Renderscript);
```

Comment:
Can one schedule to RenderScript using the gpu_ primitives? If not, then RenderScript should not be added here.